spec/system/kudo_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Kudo check', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let!(:employee) { create(:employee) }
+
+  it 'crud kudo' do
+    visit root_path
+    click_link 'Log In'
+    fill_in 'Email', with: employee.email
+    fill_in 'Password', with: employee.password
+    click_button 'Log in'
+    expect(page).to have_content 'Signed in successfully.'
+
+    click_link 'New Kudos'
+    fill_in 'Title', with: 'title test'
+    fill_in 'Content', with: 'content test'
```

Comment:
Instead of using hardcoded values I would suggest using `attributes_for` helper offered by factory bot

Reply:
I used a factory